src/SourceBuild/tarball/patches/format/0002-Add-explicit-System.Text.Json-dependency.patch

```diff
@@ -0,0 +1,73 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Mon, 26 Jun 2023 23:46:46 +0000
+Subject: [PATCH] Add explicit System.Text.Json dependency
+
+Adding an excplicit System.Text.Json dependency to prevent transitive dependency to be picked up
+from PSB, via Microsoft.Build package.
+
+Backport: https://github.com/dotnet/source-build/issues/3540
```

Comment:
This is essentially a self-referencing backport. Instead, I suggest creating PRs in format and SDK and referencing those PR URLs here.

Reply:
Yeah, certainly, will update. That's what I usually do and this was a placeholder until I had the other PRs.

Reply:
Fixed with https://github.com/dotnet/installer/pull/16826/commits/b3aeef167ccdc0a3b863aa6a5fb2826d2ababfc9